railgun/internal/ctrlutils/ingress-status.go

```diff
@@ -27,7 +27,7 @@ import (
 )
 
 const (
-	status_update_retry = 3
+	statusUpdateRetry = 3
```

Comment:
```suggestion
	STATUS_UPDATE_RETRY = 3
```

Reply:
This was intentionally changed: `golangci-lint` fails on variables that contain underscores.

Reply:
at least, we never get this syntax error complain in current CI.

Reply:
> at least, we never get this syntax error complain in current CI.

This is true, but the reason was because we had linters in CI disabled for the `railgun/` directory (which this PR fixes)

Reply:
Talked with @ccfishk in Slack, we're all set